drivers/dcf77/include/dcf77_params.h

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_dcf77
+ *
+ * @{
+ * @file
+ * @brief       Default configuration for DCF77 devices
+ *
+ * @author      Michel Gerlach <michel.gerlach@haw-hamburg.de>
+ */
+
+#ifndef DCF77_PARAMS_H
+#define DCF77_PARAMS_H
+
+#include "board.h"
+#include "dcf77.h"
+#include "saul_reg.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Set default configuration parameters for the DCF77 devices
+ * @{
+ */
+#ifndef DCF77_PARAM_PIN
+#define DCF77_PARAM_PIN               (GPIO_PIN(PB, 22))
```

Comment:
The `PB` define is not available on every platform. This causes a lot of CI errors.

`GPIO_PIN(1, 22)` should have the same effect, but will build everywhere.